tests/drives_tests.cpp

```diff
@@ -53,8 +53,18 @@ std::string run_Set_Label(char const * const input, bool cdrom) {
 
 namespace {
 
+TEST(WildFileCmp, wild_match)
+{
+    EXPECT_EQ(true, wild_match("TEST", "*"));
+    EXPECT_EQ(true, wild_match("TEST", "T*"));
+    EXPECT_EQ(true, wild_match("TEST", "T*T"));
+    EXPECT_EQ(true, wild_match("TEST", "TES?"));
+    EXPECT_EQ(false, wild_match("TEST", "Z*"));
```

Comment:
Can you add some that go beyond max length to test the error handling (or whatever we expect beyond  the 8.3 limit?)

Reply:
There is no max length for the "wild_match" function, but I am now adding more unit tests with longer names than the existing ones.